app/controllers/home_controller.rb

```diff
@@ -1,6 +1,21 @@
 class HomeController < ApplicationController
   helper PostsHelper
   def index
-    @posts = Post.order('created_at').last(3).reverse
+    # Select 3 posts within last 2 days with highest trending score
+    day_limit = 2
+    num_post = 3
+    @posts = Post
+             .select { |post| post['created_at'] >= Time.zone.today - day_limit.day }
+             .sort_by(&:compute_trending_score)
```

Comment:
Can you create an issue to optimize this later? Currently it would run this for all posts in the last 2 days. We should ideally cache this value.